src/SWP/Bundle/UserBundle/DependencyInjection/Configuration.php

```diff
@@ -58,6 +68,16 @@ public function getConfigTreeBuilder()
                                             ->scalarNode('object_manager_name')->defaultValue(null)->end()
                                         ->end()
                                     ->end()
+                                    ->arrayNode('resetpasswordrequest')
```

Comment:
name it with an underscore, please. We are using the underscore notation elsewhere so it will be consistent

Reply:
done